### PR TITLE
Blog app - Views

### PR DESCRIPTION
**Pull request by BahirHakimy**
# In this branch I have added the following
- Implemented the design from the sneak peek wireframes
- Created views for all actions

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<%= render partial: 'users/user', locals:{user:@user} -%>
+<div class="user-bio">
+  <label class="bio-label">bio</label>
+  <p><%= @user.bio -%></p>
+</div>
+<h3>Posts</h3>
+<% @user.posts.each do |post| %> <%= render partial: 'posts/post',
```
Comment (Lucas-Masaba) [inline app/views/users/show.html.erb]: - [ ] Here, you are supposed to display the most recent posts using the method you created in your user model. It should be `@users.most_recent.each`

File: app/views/posts/index.erb [hunk 2]
```diff
@@ -1,112 +1,6 @@
...
-        </div>
-        <button class="show-comments-button">Show comments</button>
-      </li>
-      <% end %>
-    </ul>
-  </body>
-</html>
+<h1>List of Posts For User <%= @user.id %></h1>
+<%= render partial: 'users/user', locals:{user:@user}%> 
+<% @posts.each do |post|%>
```
Comment (Lucas-Masaba) [inline app/views/posts/index.erb:3]: - [ ] Here, you are supposed to display the most recent comments using the method you created in your posts model. It should be `@users.most_recent_comments.each`

File: app/views/users/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+locals:{post:post} -%> <% end %>
+<button class="show-all-posts-button">Show all posts</button>
```
Comment (Lucas-Masaba) [inline app/views/users/show.html.erb]: - [ ] When I click on this button, it does not redirect to the page that shows all the users posts. Please make sure that when a user clicks on this button, you are redirected to a page that looks similar to that seen in this [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_all_posts.png)

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @BahirHakimy  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Linter checks passing :heavy_check_mark: 
- Professional README :scroll: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @Lucas-Masaba in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by KDlamini — APPROVED** [review]
### Hi @BahirHakimy 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## Highlights 🌞

✔️  Good PR title and description
✔️  Users controller `#index` action renders all users to their corresponding Views index page
✔️  The `#recent_post` method renders the user's 3 most recent posts.
✔️  `See all Post` link opens the user's post index as displayed on the wireframe
✔️  Post show page displays post with all comments

# Status: Approved ✔️ 

LeonardoInGatsby

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**